js/index.js

```diff
@@ -40,3 +40,64 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// let navigation = document.querySelectorAll('a');
+// navigation.forEach(item => item.style.color = 'green');
+
+let navItems = document.querySelector('nav').children;
+navItems[0].textContent = siteContent['nav']['nav-item-1']
+navItems[1].textContent = siteContent['nav']['nav-item-2']
+navItems[2].textContent = siteContent['nav']['nav-item-3']
+navItems[3].textContent = siteContent['nav']['nav-item-4']
+navItems[4].textContent = siteContent['nav']['nav-item-5']
+navItems[5].textContent = siteContent['nav']['nav-item-6']
+
+let navParent = document.querySelector('nav');
+let navChild = document.createElement('a');
+navChild.textContent = "link1";
+let newNavChild = document.createElement('a');
+newNavChild.textContent = 'link2';
+navParent.appendChild(navChild);
+navParent.prepend(newNavChild);
+
+
+
+let navigation = document.querySelectorAll('a');
+navigation.forEach(item => item.style.color = 'green');
+
+let banner = document.querySelector('h1');
+banner.textContent = siteContent['cta']['h1'];
+
+
+let bannerImg = document.getElementById('cta-img');
+bannerImg.src = siteContent['cta']['img-src'];
+
+let bannerBtn = document.querySelector('.cta-text button');
+bannerBtn.textContent = siteContent['cta']['button'];
+
+let h4 = document.querySelectorAll('h4');
+h4[0].textContent = siteContent['main-content']['features-h4'];
+h4[1].textContent = siteContent['main-content']['about-h4'];
+h4[2].textContent = siteContent['main-content']['services-h4'];
+h4[3].textContent = siteContent['main-content']['product-h4'];
+h4[4].textContent = siteContent['main-content']['vision-h4'];
+
+let middleImg = document.getElementById('middle-img');
+middleImg.src = siteContent['main-content']['middle-img-src'];
+console.log(middleImg);
+
+let p = document.querySelectorAll('p');
+p[0].textContent = siteContent['main-content']['features-content'];
+p[1].textContent = siteContent['main-content']['about-content'];
+p[2].textContent = siteContent['main-content']['services-content'];
+p[3].textContent = siteContent['main-content']['product-content'];
+p[4].textContent = siteContent['main-content']['vision-content'];
```

Comment:
Here is an example of how to do a for each so you don't have to write out each one:

let textContentList = document.querySelectorAll(".text-content");

textContentList.forEach((link,i) => {
  link.style.color = "green";
})


// textContentList[0].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["features-h4"];
// textContentList[0].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["features-content"];
// textContentList[1].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["about-h4"];
// textContentList[1].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["about-content"];
// textContentList[2].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["services-h4"];
// textContentList[2].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["services-content"];
// textContentList[3].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["product-h4"];
// textContentList[3].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["product-content"];
// textContentList[4].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["vision-h4"];
// textContentList[4].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["vision-content"];